examples/power_distribution.py

```diff
@@ -126,7 +127,9 @@ async def run(self) -> None:
                 if (datetime.now(timezone.utc) - time_data).total_seconds() > 30:
                     _logger.error("Active power data are stale")
                     continue
-                queue.put_nowait(active_power.value)
+                queue.put_nowait(
+                    None if not active_power.value else active_power.value.base_value
```

Comment:
The `base_value` is really just for the generic cases described here:  https://github.com/frequenz-floss/frequenz-sdk-python/pull/422#discussion_r1232042848

And for the dimensionless cases like reading from the resampler directly.

I guess the `power_distribution` example is no longer relevant anyway, because people should use the battery pool instead.  I have an in-progress branch for that,  so won't forget.  But will do it after this one is merged.